acid/features/history/tests/test_model.py

```diff
@@ -88,7 +88,7 @@ def test_build_number_return_none_if_no_build_number(self, make_buildset):
     def test_get_branches_return_branches(self, make_buildset):
         make_buildset(branch='master')
         make_buildset(branch='gimp')
-        branches = list(ZuulBuildSet.get_branches())
+        branches = list(ZuulBuildSet.get_all_branches('periodic-nightly'))
```

Comment:
Please check CI, I see this test fails from a few commits (but who use `get_all_branches`?). If it's needed, use named argument.